cedar-policy/CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ Cedar Language Version: 2.1.3
   policy. With this fix it is possible to create a link with a policy id
   after previously failing to create that link with the same id from a static
   policy.
+- Fixes an issue where certain cedar policies couldn't be converted to JSON (#601)
```

Comment:
I don't think this should be added to the 2.4.3 changelog (probably accidental due to merge)

Reply:
I'll take the opposite position 😄 I think we should add an entry for every bug fix backported (#601, #626, and #628). Also, I think this entry would be more clear as "Fixes an issue where Cedar policies containing `-i64::min` couldn't be converted to JSON"

Reply:
I'll have a PR to clean up change log after all fixes have been backported.